### ISSUE TITLE
Learning to Rank in ranking

### ISSUE THREAD

**Issue by tangzhy** [tensorflow/recommenders#181]
Hi, i wonder how to implement LTR in ranking task. Specifically, I try to incorporate point-wise and pair wise loss in the task. 

Do you have any tutorial on this topic? For now, I see the basic tutorial focuses on point-wise regression.

**Comment by maciejkula** [comment]
You can absolutely do this, and we have an upcoming tutorial that shows how to do this.

In a nutshell, you can import any [ranking loss from TensorFlow Ranking](https://github.com/tensorflow/ranking/blob/master/tensorflow_ranking/python/keras/losses.py), and use it in a `RankingTask` here. You will have to reshape your data so that predictions are 2D: the first dimension being the batch size, and the second the predictions for each candidate for a given query.
